### PR TITLE
Add tests for filesystem debug get-metadata

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: #274

### CODE REVIEW

File: testlib/dbus.py [hunk 1]
```diff
@@ -499,6 +499,28 @@ def pool_get_metadata(pool_path, *, current=True):
...
+        :rtype: The D-Bus types s, q, and s
+        :raises dbus.exceptions.DBusException:
+        """
+        iface = dbus.Interface(
+            StratisDbus._BUS.get_object(StratisDbus._BUS_NAME, pool_path),
+            StratisDbus._POOL_IFACE,
+        )
+
+        fsname_spec = (False, "") if fs_name is None else (True, fs_name)
+
```
Comment (mulkieran) [inline testlib/dbus.py:519]: No need to have a separate ```fs_name``` identifier, since you use the value only once.

File: stratis_cli_cert.py [hunk 2]
```diff
@@ -1209,6 +1209,27 @@ def test_filesystem_mount_and_write(self):
...
+        """
+        pool_name = make_test_pool(StratisCliCertify.DISKS[0:1])
+        filesystem_name = make_test_filesystem(pool_name)
+        self._unittest_command(
+            [
+                _STRATIS_CLI,
+                "filesystem",
+                "debug",
+                "get-metadata",
+                f"{pool_name}",
```
Comment (mulkieran) [inline stratis_cli_cert.py:1225]: ```^f"{pool_name}"^pool_name```

since ```pool_name``` is a string already.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Just a couple of changes, then it should be ready.

File: testlib/dbus.py (hunk 1 above)
Comment (bgurney-rh) [reply testlib/dbus.py:519]: Resolved.

File: stratis_cli_cert.py (hunk 2 above)
Comment (bgurney-rh) [reply stratis_cli_cert.py]: Resolved.

File: testlib/dbus.py (hunk 1 above)
Comment (mulkieran) [reply testlib/dbus.py:519]: You can condense it more to be consistent with other code, so just put the whole expression, ```(False,  ...) ...(True, fs_name)``` in the place of the ```fs_name``` argument in the FilesystemMetadata function call.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Last request...

**Review by mulkieran — APPROVED** [review]
Looks good. Please squash and we can merge.

**Comment by bgurney-rh** [comment]
OK; squashed into a single commit.

**Pull request merged**